src/components/calendarpage.jsx

```diff
@@ -131,13 +139,34 @@ export function CalendarPage() {
     periodTracker.createPeriodEntry(periodEntryData);
   }
 
+  const DayCell = ({ info }) => {
+    // Check if date overlaps with a period
+    for (let i = 0; i < periods.length; i++) {
+      const period = periods[i];
+      const dayCellDate = info.date;
+      const startDate = stringToDate(period.startDate);
+      const endDate = stringToDate(period.endDate);
+      if (dayCellDate >= startDate && dayCellDate <= endDate) {
+        return (<div></div>);
+      }
+    }
+    return (
+      <div>
+        {info.dayNumberText}
+      </div>
+    );
+  }
+
   const EventItem = ({ info }) => {
     const { event } = info;
     const flowType = event.extendedProps.flowType;
+    const date = toUTCDate(event._instance.range.start);
```

Comment:
Note: I think the '_' is indicating that the instance is meant to be private.
Are you sure there isn't an accessor for this?

Not a blocker

Reply:
Fixed